cmd/serverless/main.go

```diff
@@ -183,14 +185,13 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 		// execution to be stopped. TODO(remy): discuss with AWS if there is way
 		// of reporting non-critical init errors.
 		// serverless.ReportInitError(serverlessID, serverless.FatalNoAPIKey)
-		log.Error("No API key configured, exiting")
+		log.Error("No API key configured")
```

Comment:
I figured this makes more sense, as we never actually exit

Reply:
maybe we should `exit(1)` as there is no point of wasting cpu/memory?